INN/umbrella-ipbs
# Pre-footer and footer

**Should be reviewed after #53** 

## Changes

This pull request makes the following changes:

- Adds styling for the site footer and pre-footer areas

Desktop:
![screencapture-ipbs-test-2020-01-27-09_35_37](https://user-images.githubusercontent.com/18353636/73183326-9edb8b80-40e8-11ea-95a2-92516c77f729.png)

Mobile:
![screencapture-ipbs-test-2020-01-27-09_35_51](https://user-images.githubusercontent.com/18353636/73183324-9edb8b80-40e8-11ea-9405-29cfb1f0dd78.png)

## Why



For #46 and #45

## Testing/Questions

Features that this PR affects:

- Footer
- Prefooter


Questions that need to be answered before merging:


Steps to test this PR:

Pre-footer:
1. Include an additional widget region ("sidebar") just above the site footer region
2. Install and activate the [logo slider plugin](https://wordpress.org/plugins/tc-logo-slider/)
3. Add 5-6 logos to the logo slider plugin
4. In `Logo Settings` in the logo slider plugin, use these settings:
Basic settings:
 - auto play: yes
 - stop on hover: no
 - carousel loop: yes
 - slideBy items number: 1
 - items in large desktops: 4
 - items in desktops: 4
 - items in tablets: 3
 - items in mobile: 3

Advanced settings:
 - show navigation: no
 - show dots: yes
 - crop logo: no

General styling:
 - dots color: `#a8c3e3`
 - dots hover color: `#195187`

5. In the `Before Footer` widget area, add a text widget with `Sponsors` as the title and copy and paste this text:
```
Qui offici deserunt mollit anim id est laborum. Temporibus autem quisbusdam et aut officiis debitis aut rerum necessitatibus.

[tc-logo-slider]
```
6. View on the frontend and verify it looks and functions ok

Footer:
2. In theme options, select the narrow wide narrow footer layout option
3. Create a new menu with the names of `our stations` in the screenshot here https://user-images.githubusercontent.com/48263361/72483796-d7659600-37c7-11ea-838d-42b190c11b01.png
4. Create another menu for quick links
5. In the `Footer 1` widget area, add a text widget with `Our Affiliates` as the title. Content tbd
6. In the `Footer 2` widget area, add a navigation widget with `Our Stations` as the title and select your stations menu
7. In the `Footer 3` widget area, add a navigation widget with `Quick Links` as the title and select your quick links menu
8. View on the frontend and verify it looks ok
